pyiron_base/_tests.py

```diff
@@ -47,35 +72,10 @@ def tearDownClass(cls):
             pass
 
 
-class TestWithCleanProject(TestWithProject, ABC):
+class TestWithCleanProject(TestWithProject):
```

Comment:
```suggestion
class TestWithCleanProject(TestWithProject, ABC):
```

We still expect this to be a parent for individual testing suites.